endercore/src/main/java/com/enderio/core/common/network/ClientToServerMenuPacket.java

```diff
@@ -53,4 +64,8 @@ public void to(MSG packet, FriendlyByteBuf buf) {
             packet.write(buf);
         }
     }
+
+    protected void handleWrongPlayer(NetworkEvent.Context context) {
+        //TODO: handle Packet Message
```

Comment:
Could you expand on this todo so any future contributor will know what it means, thanks :)

Reply:
Thanks :)